.editorconfig

```
@@ -1,2 +1,60 @@
 # This file is the top-most EditorConfig file
-root = true
\ No newline at end of file
+root = true
+[*.cs]
+
+# CS0162: Remove unreachable code
+dotnet_diagnostic.CS0162.severity = error
```

```suggestion
dotnet_diagnostic.CS0162.severity = warning
```

Can it be warning instead? It was proven to be annoying during debugging.